python/pyspark/sql/functions.py

```diff
@@ -10018,6 +10019,220 @@ def hours(col: "ColumnOrName") -> Column:
     return _invoke_function_over_columns("hours", col)
 
 
+@try_remote_functions
+def make_dt_interval(
+    days: Optional["ColumnOrName"] = None,
+    hours: Optional["ColumnOrName"] = None,
+    mins: Optional["ColumnOrName"] = None,
+    secs: Optional["ColumnOrName"] = None,
+) -> Column:
+    """
+    Make DayTimeIntervalType duration from days, hours, mins and secs.
+
+    Examples
+    --------
```

Comment:
would you mind adding a few more examples here?

e.g. `df.select(make_dt_interval(days=lit(1), mins=lit(30)).alias('r'))`

Reply:
ok